docs/technical-documentation.md

```diff
@@ -0,0 +1,119 @@
+<br />
+
+## Initialization
+
+The Redemptions app is initialized by passing a `Vault _vault` and `TokenManager _tokenManager` parameters. Users are then able to add tokens to Redemptions via the GUI in the DAO. This passes an Ethereum contract address to the `address[] _redemptionTokenList`array. This then allows users to redeem tokens associated to the `_tokenManager` in exchange for a proportional share of each token on the `_redemptionTokenList` held in the `_vault` address.
```

Comment:
Yeah you're right. That's just a terrible paragraph lol. I'm honestly not sure how that even got in there. My bad

Reply:
Reading through this again and fixing it, and I wonder if I messed up a branch merge or something because it's really like there's just two different paragraphs here. It's just so bad... lol